Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3085,6 +3089,15 @@ def to_context(self):
                 relations_context = [relation.to_context() for relation in self.relations if relation.to_context()]
                 endpoint_context['Relations'] = relations_context
 
+            if self.vendor:
+                endpoint_context['Vendor'] = self.vendor
+
+            if self.status:
+                endpoint_context['Status'] = self.status
+
+            if self.is_isolated:
+                endpoint_context['IsIsolated'] = self.is_isolated
```

Comment:
these 2 can contain only a closed set of strings, correct? if so create a validator and raise an error otherwise, for the following devs that will implement this class.

Reply:
currently yes, it might change when adding more integrations
ok